fineftp-server/CMakeLists.txt

```diff
@@ -1,17 +1,12 @@
-cmake_minimum_required(VERSION 3.5.1)
```

Comment:
Isn't this something you are supposed to do? This CMakeLists.txt acts as top-level file, when using fineFTP in another Project as subproject.